.github/workflows/htmcore.yml

```diff
@@ -93,10 +93,12 @@ jobs:
         path: build/Release/distr/dist
 
     - name: Test python package is working
+      shell: bash
```

Comment:
nice, this forces windows to act compatible with unixes, right? (is it using the WSL?)